dannce/engine/generator.py

```diff
@@ -1816,7 +2375,23 @@ def __data_generation(self, list_IDs_temp):
 
 
 class DataGenerator_3Dconv_multiviewconsistency(keras.utils.Sequence):
-    """Generate 3d conv data from memory, with a multiview consistency objective"""
+    """Generate 3d conv data from memory, with a multiview consistency objective
+
+    Attributes:
+        batch_size (int): Description
+        chan_num (int): Description
+        data (TYPE): Description
+        expval (TYPE): Description
+        indexes (TYPE): Description
+        labels (TYPE): Description
+        list_IDs (TYPE): Description
+        nvox (TYPE): Description
+        random (TYPE): Description
+        rotation (TYPE): Description
+        shuffle (TYPE): Description
+        var_reg (TYPE): Description
+        xgrid (TYPE): Description
+    """
```

Comment:
copy from above. or actually maybe delete this whole class, because it can't be accessed from the interface right now. good to have for later, though. But we can just grab it from the git history when needed.